docs/getting-started/dev-portal/react.md

```diff
@@ -4,17 +4,21 @@ Learn how to quickly get started with Elements Dev Portal in a React project.
 
 ## Create React App Template
 
-We've created a Create React App template, which allows you to create a brand new Elements Dev Portal website in React without any additional setup.
+We've created a [Create React App template](https://github.com/stoplightio/cra-template-elements-dev-portal), which allows you to create a brand new Elements Dev Portal website in React without any additional setup.
 
-Simply run:
-
-`npx create-react-app my-dir --template @stoplight/elements-dev-portal`.
+```bash
+npx create-react-app my-dir --template @stoplight/elements-dev-portal
+```
 
 Then run `cd my-dir` and `yarn start` and you will see a basic Elements Dev Portal website in the browser.
 
-## Usage
+The project loaded is based on `https://github.com/stoplightio/studio-demo`, but you can follow the instructions below to change your `projectId` and load up your own projects.
```

Comment:
Maybe it's my English skills, but I had to literally read this sentence a few times to understand what it meant. A seemingly random GitHub link might confuse people even more.

Maybe sth:

"In order to display your own documentation, you need to change `projectId` in the generated code. Follow the instructions below to find out how."

?

Reply:
That was nonsense. Fixed it!